uvm/js/common/reports/view/Entry.js

```diff
@@ -156,32 +156,66 @@ Ext.define('Ung.view.reports.Entry', {
                     },
                     handler: 'dashboardAddRemove'
                 }, {
-                    xtype: 'exporterbutton',
-                    itemId: 'exportCsv',
+                    xtype: 'button',
                     text: 'Export Data (csv)'.t(),
+                    itemId: 'exportData',
                     iconCls: 'fa fa-external-link-square',
-                    hidden: true,
-                    disabled: true,
-                    component: this,
-                    store: this,
+                    menu: {
+                        plain: true,
+                        showSeparator: false,
+                        items: [{
+                            itemId: 'exportDatabase',
+                            text: 'Export Database'.t(),
```

Comment:
does the VM have context here of the type of events we are exporting to use in this text?